tests/ops/fixtures/saas/fullstory_fixtures.py

```diff
@@ -0,0 +1,283 @@
+    fides_key = fullstory_config["fides_key"]
+    connection_config = ConnectionConfig.create(
+        db=db,
+        data={
+            "key": fides_key,
+            "name": fides_key,
+            "connection_type": ConnectionType.saas,
+            "access": AccessLevel.write,
+            "secrets": fullstory_secrets,
+            "saas_config": fullstory_config,
+        },
+    )
+    yield connection_config
+    connection_config.delete(db)
+
+
+@pytest.fixture
+def fullstory_dataset_config(
+    db: Session,
+    fullstory_connection_config: ConnectionConfig,
+    fullstory_dataset: Dict[str, Any],
+) -> Generator:
+    fides_key = fullstory_dataset["fides_key"]
+    fullstory_connection_config.name = fides_key
+    fullstory_connection_config.key = fides_key
+    fullstory_connection_config.save(db=db)
+    dataset = DatasetConfig.create(
+        db=db,
+        data={
+            "connection_config_id": fullstory_connection_config.id,
+            "fides_key": fides_key,
+            "dataset": fullstory_dataset,
+        },
+    )
+    yield dataset
+    dataset.delete(db=db)
+
+
+@pytest.fixture()
+def fullstory_postgres_dataset() -> Dict[str, Any]:
+    return {
+        "fides_key": "fullstory_postgres",
+        "name": "Fullstory Postgres",
+        "description": "Lookup for Fullstory User IDs",
+        "collections": [
+            {
+                "name": "fullstory_users",
+                "fields": [
+                    {
+                        "name": "email",
+                        "data_categories": ["user.contact.email"],
+                        "fidesops_meta": {"data_type": "string", "identity": "email"},
+                    },
+                    {
+                        "name": "fullstory_user_id",
+                        "fidesops_meta": {"data_type": "string"},
+                    },
+                ],
+            }
+        ],
+    }
+
+
+@pytest.fixture
+def fullstory_postgres_dataset_config(
+    connection_config: ConnectionConfig,
+    fullstory_postgres_dataset: Dict[str, Any],
+    db: Session,
+) -> Generator:
+    fides_key = fullstory_postgres_dataset["fides_key"]
+    connection_config.name = fides_key
+    connection_config.key = fides_key
+    connection_config.save(db=db)
+    dataset = DatasetConfig.create(
+        db=db,
+        data={
+            "connection_config_id": connection_config.id,
+            "fides_key": fides_key,
+            "dataset": fullstory_postgres_dataset,
+        },
+    )
+    yield dataset
+    dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def fullstory_postgres_db(postgres_integration_session):
+    postgres_integration_session = seed_postgres_data(
+        postgres_integration_session,
+        "./tests/ops/fixtures/saas/external_datasets/fullstory.sql",
+    )
+    yield postgres_integration_session
+    drop_database(postgres_integration_session.bind.url)
+
+
+class FullstoryTestClient:
+
+    headers: object = {}
```

Comment:
We can get rid of `headers` and `base_url` since we're defining these in the `__init__`